src/code/z_actor.c

```diff
@@ -2469,13 +2469,17 @@ void Actor_FreezeAllEnemies(GlobalContext* globalCtx, ActorContext* actorCtx, s3
     }
 }
 
-void func_80031B14(GlobalContext* globalCtx, ActorContext* actorCtx) {
+void Actor_ChangeRoom(GlobalContext* globalCtx, ActorContext* actorCtx) {
     Actor* actor;
     s32 i;
 
     for (i = 0; i < ARRAY_COUNT(actorCtx->actorLists); i++) {
         actor = actorCtx->actorLists[i].head;
         while (actor != NULL) {
+            //! @bug Stale Reference Manipulation.  Under very specific circumstances an actor can be culled and held by
+            //! the player.  Because of this its room is never set to -1 causing it to be deallocated during a room
```

Comment:
I don't think we double space after punctuation anywhere else in the repo, should I wouldn't do that in this comment